deploy/kubernetes/dolphinscheduler/Chart.yaml

```diff
@@ -35,11 +35,11 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 2.0.0
+version: 2.0.1
```

Comment:
Is it a unrelated change?

Reply:
The comment says we should increment `version` if we change `app version`. Did I get it wrong?
![image](https://user-images.githubusercontent.com/34905992/201569308-3a4cffaa-b379-46f7-8e35-74c4dfea33b7.png)